pro/src/screens/SignupJourneyForm/Offerers/__specs__/Offerers.spec.tsx

```diff
@@ -396,19 +396,13 @@ describe('screens:SignupJourney::Offerers', () => {
       },
     }
 
-    it('Should display the new banner when the FF is enabled', async () => {
+    it('Should not display join button when the FF is enabled', async () => {
```

Comment:
tu pourrais en profiter pour expliciter "when the FF is enabled" - peut-être quelque chose comme "when display banner about the possibility to be invited by a colleague is enabled" (c'est un peu long, tu vois peut-être comment faire clair et + court ?)